save-sandbox/db/tables/execution.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+
+    <changeSet id="execution-1" author="frolov" context="dev or prod">
+        <createTable tableName="execution">
+            <column name="id" type="bigint" autoIncrement="true">
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="start_time" type="timestamp">
+                <constraints nullable="false"/>
+            </column>
+            <column name="end_time" type="timestamp">
+                <constraints nullable="true" />
+            </column>
+            <column name="status" type="varchar(100)">
+                <constraints nullable="false" />
+            </column>
+            <column name="resources_root_path" type="varchar(255)">
```

Comment:
There is no such field in Execution

Reply:
This field is in Execution. If it is not needed, it must be removed.

![image](https://user-images.githubusercontent.com/50615459/193002226-46920306-1e27-45c6-8999-f4e350011c5c.png)

Reply:
I see, looks like we forget to remove it from database: `com.saveourtool.save.entities.Execution` doesn't contain it.

Can you please remove it?

Reply:
Yes. Removed it